packages/website/ts/blockchain.ts

```diff
@@ -92,16 +94,65 @@ export class Blockchain {
         }
         return providerNameIfExists;
     }
-    private static async _getProviderAsync(injectedWeb3: Web3, networkIdIfExists: number): Promise<Provider> {
+    private static _getInjectedWeb3(): any {
+        return (window as any).web3;
+    }
+    private static _getFallbackNetworkId(): number {
+        return configs.IS_MAINNET_ENABLED ? constants.NETWORK_ID_MAINNET : constants.NETWORK_ID_KOVAN;
```

Comment:
We can probably remove `IS_MAINNET_ENABLED`, I think it was just a feature flag for before 0x was deployed on mainnet